doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -2992,9 +2994,60 @@ Deletes an ``AuthenticatedUser`` whose ``id``  is passed. ::
 
     DELETE http://$SERVER/api/admin/authenticatedUsers/id/$id
     
-Note: If the user has performed certain actions such as creating or contributing to a Dataset or downloading a file they cannot be deleted.
+Note: If the user has performed certain actions such as creating or contributing to a Dataset or downloading a file they cannot be deleted. If a user cannot be deleted due to performing certain actions, you can choose to :ref:`disable a user<disable-a-user>`.
```

Comment:
seems redundant to say "performing certain actions" again. Make "for this reason"?

Reply:
sounds good, addressed in https://github.com/IQSS/dataverse/pull/7585/commits/2af33fd9662328d86518c6f8c985c187398d1a0b